cli/send.go

```diff
@@ -143,6 +148,21 @@ var sendCmd = &cli.Command{
 			Params:     params,
 		}
 
+		// Funds insufficient check
+		fromBalance, err := api.WalletBalance(ctx, msg.From)
+		if err != nil {
+			return err
```

Comment:
Maybe ignore this error when force is passed

Reply:
if force specified will not check insufficient fund